# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1457642311 18000
#      Thu Mar 10 15:38:31 2016 -0500
# Branch stable
# Node ID 882cb9f032cef55267f4c20a94f383c5eebd8200
# Parent  2b66f30a0de7a09801c2a3566021cde7e194d3d3
Added tag rc-4-0-1-4 for changeset 2b66f30a0de7

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -92,8 +92,9 @@ 46df2c7b8f4dec400ce353370f2e47a02926026d
 a66548dc07b0fc9b7d37a4be065aa15e8243c96a release-3-8-2
 52d2bbf49c922440621b42b1c1679f0e55462485 rc-4-0-0-1
 eba80000fa0dad32ba0f1cd767dd826d1ce1aba6 rc-4-0-0-2
 065f933ef08318e40b81f7fe75236e6175088117 rc-4-0-0-3
 42bb3a776c9fcc008669f382d2409297c4a901b3 rc-4-0-0-4
 20b17dda0e0c838646dfdccec153ec8cdf6d3c37 rc-4-0-1-1
 0cfcc178432bfc7cfdef5dca96e74ec5884ba752 rc-4-0-1-2
 a58d5e6df908199c4aea83f78077b81289d2a0ad rc-4-0-1-3
+2b66f30a0de7a09801c2a3566021cde7e194d3d3 rc-4-0-1-4
